openhands/core/config/mcp_config.py

```diff
@@ -132,6 +137,9 @@ def from_toml_section(cls, data: dict) -> dict[str, 'MCPConfig']:
                     servers.append(MCPStdioServerConfig(**server))
                 data['stdio_servers'] = servers
 
+            if 'streamable_http_servers' in data:
+                pass
```

Comment:
Do we just ignore streamable http server defined in config?

Reply:
Yet to implement, doing it now!